rocAL/include/pipeline/commons.h

```diff
@@ -227,6 +237,7 @@ enum class ResizeScalingMode {
     NOT_LARGER,
     MIN_MAX
 };
+REGISTER_ENUM(ResizeScalingMode)
```

Comment:
Non-blocking: Associated enum is defined as . Consider explicitly specifying the underlying type (e.g., ) to keep ABI stable and avoid surprises across compilers.